ush/config_defaults.yaml

```diff
@@ -61,13 +61,19 @@ user:
   JOBSdir: '{{ [HOMEaqm, "jobs"]|path_join }}'
   SORCdir: '{{ [HOMEaqm, "sorc"]|path_join }}'
   PARMdir: '{{ [HOMEaqm, "parm"]|path_join }}'
+  FIXdir: '{{ [HOMEaqm, "fix"]|path_join }}'
   MODULESdir: '{{ [HOMEaqm, "modulefiles"]|path_join }}'
   EXECdir: '{{ [HOMEaqm, workflow.EXEC_SUBDIR]|path_join }}'
   VX_CONFIG_DIR: '{{ [HOMEaqm, "parm"]|path_join }}'
   METPLUS_CONF: '{{ [PARMdir, "metplus"]|path_join }}'
   MET_CONFIG: '{{ [PARMdir, "met"]|path_join }}'
   UFS_WTHR_MDL_DIR: '{{ user.UFS_WTHR_MDL_DIR }}'
   ARL_NEXUS_DIR: '{{ [SORCdir, "arl_nexus"]|path_join }}'
+  FIXaqm: '{{ [HOMEaqm, "fix"]|path_join }}' 
+  FIXaqmchem_lbcs: '{{ [FIXaqm, "chem_lbcs"]|path_join }}' 
+  FIXaqmfire: '{{ [FIXaqm, "fire"]|path_join }}' 
+  FIXaqmnexus: '{{ [FIXaqm, "nexus"]|path_join }}' 
+  FIXaqmnexus_gfs_sfc: '{{ [FIXaqm, "gfs"]|path_join }}' 
 
```

Comment:
@JianpingHuang-NOAA, can you move the above 5 lines for FIXaqmxxx to below (locate them with the other FIXaqmxxx) to prevent any duplicates?